server/controllers/calendarController.ts

```diff
@@ -16,13 +16,13 @@ export default class CalendarController {
     private readonly userService: UserService,
   ) {}
 
-  async getDate(req: Request, res: Response) {
+  async getDate(req: Request, res: Response): Promise<void> {
```

Comment:
turns out that returning `Promise<false | void>` doesn't pass the type checking.  I've therefore had to remove the `return false`